CMakeLists.txt

```diff
@@ -1,6 +1,24 @@
 # fixme: this should be 3.3, but that breaks static ncurses
 cmake_minimum_required(VERSION 3.2)
 
+set(ENABLE_CONDA OFF)
+if(DEFINED ENV{CONDA_PREFIX})
+  set(ENABLE_CONDA ON)
+  # various fixes and workarounds
+  add_definitions("-Dsecure_getenv=getenv")
+  # fixes `undefined reference to `boost::system::detail::system_category_instance'`:
+  add_definitions("-DBOOST_ERROR_CODE_HEADER_ONLY")
+  # Adding formating macros
+  add_definitions("-D__STDC_FORMAT_MACROS=1")
+  # fixes always_inline attribute errors
+  add_compile_options("-fno-semantic-interposition")
+  # Adding `--sysroot=...` resolves `no member named 'signbit' in the global namespace` error:
+  set(CMAKE_SYSROOT "$ENV{CONDA_BUILD_SYSROOT}")
+  set(CMAKE_PREFIX_PATH "$ENV{CONDA_PREFIX}")
+  # Adding `-I$BUILD_SYSROOT_INLCUDE` resolves `cassert file not found` error:
+  include_directories(SYSTEM "$ENV{CONDA_BUILD_SYSROOT}/usr/include")
```

Comment:
The comment adds `BUILD_SYSROOT_INLCUDE` but actually adding `CONDA_BUILD_SYSROOT/usr/include`. Fix comment?

Reply:
Considering https://github.com/conda-forge/omniscidb-feedstock/blob/master/recipe/build.sh, I am not sure if this is needed.